src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -57,6 +57,13 @@ private static Option2<T> CreateOption<T>(OptionGroup group, string name, T defa
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_style_prefer_switch_expression"),
                 new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(PreferSwitchExpression)}")});
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferPatternMatching = CreateOption(
+            CSharpCodeStyleOptionGroups.PatternMatching, nameof(PreferPatternMatching),
+            defaultValue: CodeStyleOption2<bool>.Default,
```

Comment:
so just to check, but this is off by default?

Or is it on, but doesn't appear unless you explicitly bring up the lightbulb?  I would be fine with eitehr approaches.

Reply:
I intended for this to be enabled with "refactoring-only" option.. Surprised to see we don't have a cached value of `new CodeStyleOption2<bool>(true, Silent)`? in that case I'll make one.